### PR TITLE
Add web server in proxy with sample handler

### PR DESCRIPTION
**Pull request by haorenfsa**
Signed-off-by: shaoyue.chen <shaoyue.chen@zilliz.com>
/cc @xiaofan-luan @congqixia @ArkaprabhaChakraborty @Anosh21

part of : #7700

### CODE REVIEW

File: internal/distributed/proxy/service.go [hunk 1]
```diff
@@ -14,20 +14,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package grpcproxy
+package proxy
```
Comment (congqixia) [inline internal/distributed/proxy/service.go:17]: I prefer naming this way while it conflicts other package's naming convention. Not sure we should change that here.

Comment (haorenfsa) [reply internal/distributed/proxy/service.go:17]: This package is quoted only in 2 position, so for compile level it's ok to change here. 
But it's sub package `proxy/client` is still named as `grpcclient` which is widely used. 
So maybe we should not change for now, and leave it for another PR focused to change them all together

**Comment by haorenfsa** [comment]
@congqixia updated

**Comment by Anosh21** [comment]
> @congqixia updated
comment rerun ut can trigger the job again, run the test again and push again.

File: internal/distributed/proxy/service.go [hunk 2]
```diff
@@ -159,6 +187,8 @@ func (s *Server) Run() error {
 func (s *Server) init() error {
 	Params.InitOnce(typeutil.ProxyRole)
 	log.Debug("Proxy init service's parameter table done")
+	HTTPParams.Init()
+	log.Debug("Proxy init http server's parameter table done")
```
[the remark this answers is not in the document]
Comment (haorenfsa) [reply internal/distributed/proxy/service.go:191]: log added below when http.enabled

**Comment by Anosh21** [comment]
lgtm

**Comment by Anosh21** [comment]
Lgtm! @xiaofan-luan

**Review by congqixia — APPROVED** [review]
/lgtm

**Comment by congqixia** [comment]
BTW, I was wondering that if you could kindly create an issue for the feature and post the current design in it? #7700 is just a feature request and there is no design detail in it. @haorenfsa

**Comment by Anosh21** [comment]
we can edit the title #7700  and add the design feature too. @congqixia

**Comment by congqixia** [comment]
> we can edit the title #7700 and add the design feature too. @congqixia

@Anosh21 Sounds great. Just one thing I shall mention that maybe we does not has the privileged to edit someone else's comment. So either a new issue or pending comments in the original feature request will do.

**Comment by haorenfsa** [comment]
> BTW, I was wondering that if you could kindly create an issue for the feature and post the current design in it? #7700 is just a feature request and there is no design detail in it. @haorenfsa

yes, will do later

**Pull request merged**